src/state/reducers/infoResponses.js

```diff
@@ -42,6 +44,13 @@ export const infoResponsesReducer = (state = {}, action) => {
       }, {});
     case ActionTypes.IMPORT_MIRADOR_STATE:
       return {};
+    case ActionTypes.RESET_AUTHENTICATION_STATE:
```

Comment:
The info response for the given tokenService is now removed. A user canvas navigation will retrigger this to be requested. We could do other things, but maybe this is ok?